core/src/avm2/globals/flash/net/URLVariables.as

```diff
@@ -0,0 +1,38 @@
+package flash.net {
+    import flash.utils.escapeMultiByte;
+    public dynamic class URLVariables {
+	// TODO: construct from String
+        private var _url: String;
+	public function URLVariables(url:String = null) {
+	    this._url = url;
+	}
+	public function set url(value:String): void {
```

Comment:
I don't see any `url` property in the [docs](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/net/URLVariables.html). I do see however `decode()`, which is missing in this PR.

Reply:
I added a check and a TODO to make it clear what is not implemented yet (basically it currently works as an encoder but not as a decoder).

Reply:
Update: had to remove the check because the linter complained. Left TODOs.